cmd/gh/main.go

```diff
@@ -56,6 +56,13 @@ func mainRun() exitCode {
 
 	ctx := context.Background()
 
+	if cfg, err := cmdFactory.Config(); err == nil {
+		if err := cfg.MigrateMultiAccount(); err != nil {
```

Comment:
Thoughts about removing the knowledge of `MultiAccount` from `MigrateMultiAccount`?

Reply:
Is mostly the same comment as https://github.com/cli/cli/pull/8281/files#r1378891821? I am on board with that change.

Reply:
Yes, becomes `Migrate` and then the `main` is responsible for injecting the `migration.MultiAccount`